Makefile

```diff
@@ -306,9 +331,11 @@ publish-release:
 dep-graph:
 	cargo deps --optional-deps --filter wasmer-wasi wasmer-wasi-tests wasmer-kernel-loader wasmer-dev-utils wasmer-llvm-backend wasmer-emscripten wasmer-emscripten-tests wasmer-runtime-core wasmer-runtime wasmer-middleware-common wasmer-middleware-common-tests wasmer-singlepass-backend wasmer-clif-backend wasmer --manifest-path Cargo.toml | dot -Tpng > wasmer_depgraph.png
 
-docs:
-	cargo doc --features=backend-singlepass,backend-cranelift,backend-llvm,docs,wasi,managed --workspace --document-private-items --no-deps
+docs-capi:
 	cd lib/runtime-c-api/ && doxygen doxyfile && cd ..
```

Comment:
You didn't write this code in this PR, but it looks wrong, we cd 2 directories up and 1 directory back.

`cd -` pops back to where you `cd`'d from no matter where it was!